File: Summaryzer/app/src/main/java/com/app/summaryzer/ImageCofirmation.java
package com.app.summaryzer;

import android.content.Intent;
import android.content.SharedPreferences;
import android.graphics.BitmapFactory;
import android.os.Bundle;
import android.widget.ImageButton;
import android.widget.ImageView;

import androidx.appcompat.app.AppCompatActivity;

public class ImageCofirmation extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        themSetter(getThemeStatus());
        setContentView(R.layout.activity_image_cofirmation);
        String imagepath = getImagePath();

        ImageView imageView = findViewById(R.id.captured_img_preview);
        imageView.setImageBitmap(BitmapFactory.decodeFile(imagepath));

        ImageButton redobtn = findViewById(R.id.retakeButt);
        redobtn.setOnClickListener(view -> finish());

        ImageButton cancelbtn = findViewById(R.id.cancelCaptureButt);
        cancelbtn.setOnClickListener(view -> {
            Intent i=new Intent(ImageCofirmation.this, MainActivity.class);
            i.setFlags(Intent.FLAG_ACTIVITY_CLEAR_TOP); // To clean up all activities
            startActivity(i);
        });

        ImageButton donebtn = findViewById(R.id.doneCaptureButt);
        donebtn.setOnClickListener(view -> {
            Intent i=new Intent(ImageCofirmation.this, TextOpenActivity.class);
            startActivity(i);
        });
    }

    private void themSetter(int tcode){
        switch (tcode){
            case 101: this.setTheme(R.style.AppTheme);break;
            case 102: this.setTheme(R.style.LightTheme);break;
            case 103: this.setTheme(R.style.joyTheme);break;
        }
    }
    private int getThemeStatus(){
        SharedPreferences mSharedPreferences = this.getSharedPreferences("theme", MODE_PRIVATE);
        return mSharedPreferences.getInt("themeCode", 0);
    }

    private String getImagePath(){
        SharedPreferences mSharedPreferences = this.getSharedPreferences("ImageCaptured", MODE_PRIVATE);
        return mSharedPreferences.getString("imagepath", "");
    }
}

File: app/src/main/java/org/ranjanistic/attorney/ImageCofirmation.kt
package org.ranjanistic.attorney

import android.content.Intent
import android.graphics.BitmapFactory
import android.os.Bundle
import android.view.View
import android.widget.ImageButton
import android.widget.ImageView
import androidx.appcompat.app.AppCompatActivity
import com.app.summaryzer.R

class ImageCofirmation : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        themSetter(themeStatus)
        setContentView(R.layout.activity_image_cofirmation)
        val imagepath = imagePath
        val imageView = findViewById<ImageView>(R.id.captured_img_preview)
        imageView.setImageBitmap(BitmapFactory.decodeFile(imagepath))
        val redobtn = findViewById<ImageButton>(R.id.retakeButt)
        redobtn.setOnClickListener { view: View? -> finish() }
        val cancelbtn = findViewById<ImageButton>(R.id.cancelCaptureButt)
        cancelbtn.setOnClickListener { view: View? ->
            val i = Intent(this@ImageCofirmation, MainActivity::class.java)
            i.flags = Intent.FLAG_ACTIVITY_CLEAR_TOP // To clean up all activities
            startActivity(i)
        }
        val donebtn = findViewById<ImageButton>(R.id.doneCaptureButt)
        donebtn.setOnClickListener { view: View? ->
            val i = Intent(this@ImageCofirmation, TextOpenActivity::class.java)
            startActivity(i)
        }
    }

    private fun themSetter(tcode: Int) {
        when (tcode) {
            101 -> this.setTheme(R.style.AppTheme)
            102 -> this.setTheme(R.style.LightTheme)
            103 -> this.setTheme(R.style.joyTheme)
        }
    }

    private val themeStatus: Int
        private get() {
            val mSharedPreferences = getSharedPreferences("theme", MODE_PRIVATE)
            return mSharedPreferences.getInt("themeCode", 0)
        }
    private val imagePath: String?
        private get() {
            val mSharedPreferences = getSharedPreferences("ImageCaptured", MODE_PRIVATE)
            return mSharedPreferences.getString("imagepath", "")
        }
}